components/FA1_2.tsx

```diff
@@ -115,7 +114,7 @@ const FA1_2 = ({ index, remove, children }: props) => {
             };
           })
           .then(res => res.json()),
-        5000
```

Comment:
will it retry if timeout? 
It just happened to me that I wait longer than 1 min ... :thinking: ?